### PR TITLE
Fix/mfs/cleanup

### PR DESCRIPTION
**Pull request by polezaivsani**
Just a few minor fixes as i went learning the code

### CODE REVIEW

File: mfs/system.go [hunk 1]
```diff
@@ -1,12 +1,6 @@
 // package mfs implements an in memory model of a mutable ipfs filesystem.
 //
-// It consists of four main structs:
-// 1) The Filesystem
```
Comment (whyrusleeping) [inline mfs/system.go]: Granted, this comment wasnt entirely accurate anymore, but i would like to have something at least as informative in its place

Comment (polezaivsani) [inline mfs/system.go]: This has just the short description what root is missing, but it's on the type itself. Thought that'll be one less chance of comments and code going out of sync.

Still want this back?

Comment (polezaivsani) [inline mfs/system.go]: The the whole thing may not warrant the attention, but still - you've seen the comment on line 3 - doesn't it serve as a good enough overview sans obsolete bits?

**Comment by whyrusleeping** [comment]
The purpose for that comment was to make pages like this (which are autogenerated)  look nice: https://godoc.org/github.com/ipfs/go-ipfs/mfs

It currently doesnt look very nice, but i do want to keep that top section informative at a glance.

**Comment by Kubuxu** [comment]
Closing due to lack of activity.

**Pull request closed without merging**